spec/ParseQuery.spec.js

```diff
@@ -3888,8 +4178,6 @@ describe('Parse.Query testing', () => {
       })
       .then(function (savedFoobar) {
         const foobarQuery = new Parse.Query('Foobar');
-        foobarQuery.include('barBaz');
-        foobarQuery.include('barBaz.bazoo');
         foobarQuery.select(['fizz', 'barBaz.key', 'barBaz.bazoo.some']);
```

Comment:
Same thing here. What we really want to test is this code: https://github.com/parse-community/parse-server/blob/0f5928542c44fb8b72ddca4a2d35c0905e2d335d/src/RestQuery.js#L100-L123

Making sure that when `exclude` is used, the correct keys are included without specifying `include`